content/webapp/server.js

```diff
@@ -1,11 +1,10 @@
 const Koa = require('koa');
 const Router = require('koa-router');
 const next = require('next');
-const Cookies = require('cookies');
 const Prismic = require('prismic-javascript');
 const linkResolver = require('@weco/common/services/prismic/link-resolver');
-const { initialize, isEnabled } = require('@weco/common/services/unleash/feature-toggles');
 const withGlobalAlert = require('@weco/common/koa-middleware/withGlobalAlert');
+const withToggles = require('@weco/common/koa-middleware/withToggles');
```

Comment:
need to add to catalogue app too

Reply:
Will do as second step as trying to keep the PR small as it's quite a lot of changes.